src/Build/Logging/FancyLogger/FancyLoggerProjectNode.cs

```diff
@@ -38,6 +38,8 @@ private static string GetUnambiguousPath(string path)
         public int MessageCount = 0;
         public int WarningCount = 0;
         public int ErrorCount = 0;
+        // Bool if node should rerender
+        internal bool ShouldRerender = true;
```

Comment:
`ShouldRerender` + `Render()` is starting to look like an `IConsoleRenderable` interface--worth thinking about whether we could simplify code by creating one.